apps/admin/src/foster-families/db.server.ts

```diff
@@ -29,11 +31,15 @@ export class FosterFamilyDbDelegate {
     // Don't use Algolia when there are no text search.
     if (displayName == null) {
       const fosterFamilies = await prisma.fosterFamily.findMany({
+        where: {
+          isBanned: isBanned ?? undefined,
```

Comment:
Tu devrais pouvoir simplifier cette ligne en :

```suggestion
          isBanned,
```